app/templates/components/warning-message.hbs

```diff
@@ -0,0 +1,5 @@
+<div class="warning">
+    <img src="assets/images/warning.png" alt="warning icon">
```

Comment:
If you can easily find an svg instead of a png then it would be great since we wouldn't have to worry about resolution. Otherwise png is fine :)